.editorconfig

```diff
@@ -0,0 +1,5 @@
+root = true
```

Comment:
ignore this file?

Reply:
This is meant to be committed, it enforces a uniform formatting for all files (like prettier, but for files prettier doesn't format) https://editorconfig.org

Reply:
It was in the CLI repo beforehand

Reply:
Got it